src/stactools/cli/commands/validate.py

```diff
@@ -15,25 +17,42 @@ def create_validate_command(cli):
     @click.option("--recurse/--no-recurse",
                   default=True,
                   help=("If false, do not validate any children "
-                        "(only useful for Catalogs and Collections"))
+                        "(only useful for Catalogs and Collections)"))
     @click.option("--links/--no-links",
                   default=True,
                   help=("If false, do not check any of the objects's links."))
     @click.option(
         "--assets/--no-assets",
         default=True,
         help=("If false, do not check any of the collection's/item's assets."))
-    def validate_command(href, recurse, links, assets):
+    @click.option("--stac-vrt",
```

Comment:
It'd be nice to generalize this, since these kinds of checks will be useful to several projects (GDAL dev now includes a driver for STAC items, so the `stac-vrt` library is basically obsolete now).

I'm not sure what name would be best, but perhaps just `--proj`?

Reply:
I can't think of anything better. It's a bit misleading because the `proj` extension only required the `proj:epsg` field. But it will be better than `--stac-vrt`.

Reply:
Would it make sense to have an enumeration of `--best-practices` and then have a bunch of named best practices validators? E.g. `proj`, `eo`, `summaries`, etc, with a default `all`? So you'd call `stac validate --best-practices proj,summaries catalog.json` or whatever.